src/sentry/static/sentry/app/views/organizationMonitors/monitorForm.jsx

```diff
@@ -0,0 +1,124 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Access from 'app/components/acl/access';
+import Field from 'app/views/settings/components/forms/field';
+import Form from 'app/views/settings/components/forms/form';
+import SelectField from 'app/views/settings/components/forms/selectField';
+import TextCopyInput from 'app/views/settings/components/forms/textCopyInput';
+import TextField from 'app/views/settings/components/forms/textField';
+import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
+import SentryTypes from 'app/sentryTypes';
+import {t, tct} from 'app/locale';
+
+import MonitorModel from './monitorModel';
+
+export default class MonitorForm extends Component {
+  static propTypes = {
+    monitor: SentryTypes.Monitor,
+    apiEndpoint: PropTypes.string.isRequired,
+    apiMethod: PropTypes.string.isRequired,
+    onSubmitSuccess: PropTypes.func.isRequired,
+  };
+
+
+  constructor(...args) {
+    super(...args);
+    const {monitor} = this.props;
+    this.form = new MonitorModel();
+    if (monitor) {
```

Comment:
You don't need to do this, passing `initialData` to `<Form>` is sufficient

Reply:
ah does Form bind initialData on model? I only tried the inverse way.